tools/build_all.sh

```diff
@@ -7,11 +7,13 @@
 # set at the time we start the last step, the ros1_bridge build.
 
 if [ "$ROS2_DISTRO" = "" ]; then
-	export ROS2_DISTRO=dashing
+  export ROS2_DISTRO=eloquent
 fi
-if [ "$ROS2_DISTRO" != "dashing" ]; then
-	echo "ROS2_DISTRO variable must be set to dashing"
-	exit 1
+if [ "$ROS2_DISTRO" != "eloquent" ]; then
```

Comment:
This can / should be just eloquent. The dashing version of this script in the dashing branch is only dashing. This should be merged into master and cherry picked into Eloquent with just eloquent.